lms/djangoapps/verify_student/models.py

```diff
@@ -641,6 +642,12 @@ def original_verification(cls, user):
         query = cls.objects.filter(user=user, window=None).order_by('-updated_at')
         return query[0]
 
+    @classmethod
+    def get_initial_verification(cls, user):
+        init_verification = cls.objects.filter(user=user, status__in=["submitted", "approved"], window=None).\
+            order_by("-created_at")[0:1]
+        return init_verification[0] if init_verification else None
```

Comment:
It would be cleaner to use [`latest()`](https://docs.djangoproject.com/en/1.4/ref/models/querysets/#latest) and [`exists()`](https://docs.djangoproject.com/en/1.4/ref/models/querysets/#exists) in this context:
```python
init_verification = cls.objects.filter(user=user, status__in=["submitted", "approved"], window=None)

return init_verification.latest('created_at') if init_verification.exists() else None
```